ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4-dev-preview.yaml

```diff
@@ -0,0 +1,116 @@
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: konveyor
+    tag: ubi9-v1.22.2
+images:
+- dockerfile_path: Dockerfile
+  to: oadp-operator
+- dockerfile_path: build/ci-Dockerfile
+  to: test-oadp-operator
+operator:
+  bundles:
+  - dockerfile_path: build/Dockerfile.bundle
+  substitutions:
+  - pullspec: quay.io/konveyor/oadp-operator:latest
+    with: oadp-operator
+releases:
+  latest:
+    prerelease:
+      product: ocp
+      version_bounds:
+        lower: 4.18.0-0
+        stream: 4-dev-preview
+        upper: 4.19.0-0
```

Comment:
what is the goal of this job?

Reply:
Dev preview of upcoming openshift releases so we have more time to catch issues like changes in 4.15 registry secret handling https://issues.redhat.com/browse/OADP-3598

Reply:
So not just candidate of next version preGA like 4.17, but prerelease of not yet release candidates like 4.18

Reply:
which version of OCP will it use to run the job?

Reply:
https://amd64.ocp.releases.ci.openshift.org/
it will find latest items within the range defined and available.. currently 4.18.0-ec.1 I am assuming.

Reply:
it is somehow automated, but I think is better to be explicit here, by changing this to use latest preview available of 4.18 (if possible) and changing file name to something with 4-18-dev-preview

My fear is that people will think this will always use next dev preview version available, but lower and upper fields will prevent that

Reply:
lower and upper is a required field. we can make upper bound 5.0 also. WDYT.

Reply:
There will be frequent renaming of this file if we include version name in the file name.

Reply:
finding someone to specify $next unreleased version as an alias would be better than having to update prow every four months.  If we HAVE to specify a version than the job name should equal the version :)